main.go

```diff
@@ -165,54 +174,75 @@ func mainE(config Configs) error {
 
 	fmt.Println()
 
-	log.Infof("Export APKs:")
+	log.Infof("Export Artifacts:")
```

Comment:
warn the user if deprecated inputs are provided

Reply:
Fixed